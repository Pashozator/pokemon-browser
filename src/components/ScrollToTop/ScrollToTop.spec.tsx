import React from 'react'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ScrollToTop from './ScrollToTop';
import { StaticRouter } from 'react-router-dom';

describe('ScrollToTop component', () => {
	it('should render', () => {
		const wrapper = shallow(
			<StaticRouter>
				<ScrollToTop/>
			</StaticRouter>
		);

		expect(wrapper.exists()).toBeTruthy();
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
