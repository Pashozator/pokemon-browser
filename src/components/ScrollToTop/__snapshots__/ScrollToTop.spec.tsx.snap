// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop component should render 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
  staticContext={Object {}}
>
  <ScrollToTop />
</Router>
`;
